osu.Game.Tournament/Screens/Editors/Components/TournamentClearAllDialog.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Overlays.Dialog;
+
+namespace osu.Game.Tournament.Screens.Editors.Components
+{
+    public partial class TournamentClearAllDialog : DangerousActionDialog
+
+    {
+        public TournamentClearAllDialog(IList storage)
+        {
+            HeaderText = @"Confirm clear all?";
```

Comment:
```suggestion
            HeaderText = @"Clear all?";
```

Mind you this text is still wonderfully unspecific but making it specific is going to be somewhat painful due to surrounding structure of `TournamentEditorScreen` so probably fine to stop at the generic option...